components/Registration/Registration.tsx

```diff
@@ -45,14 +47,13 @@ const Form: React.FC = () => {
                         {React.createElement(pages[formIndex])}
                     </form>
                 </FormProvider>
-                {formIndex != pages.length - 1 ||
-                    (true && (
-                        <Navigation
-                            index={formIndex}
-                            handlePrevious={previousPage}
-                            handleNext={nextPage}
-                        />
-                    ))}
+                {formIndex != pages.length - 1 && true && (
```

Comment:
the "true" can be removed now that we have more than one page. It was only for testing :)